simple_network_sim/network_of_populations.py

```diff
@@ -14,6 +14,17 @@
 
 # CurrentlyInUse
 def countInfectiousAgeStructured(dictOfStates, time):
+    """Count the number of infectious individuals in all states at some time.
+
+    :param dictOfStates: A time series of the disease states in each region stratified by age.
+    :type dictOfStates: A dictionary with time as keys and whose values are another dictionary with
+    the region as a key and the disease state as values. The states are a dictionary with a tuple
+    of (age, state) as keys and the number of individuals in that state as values.
+    :param time: The time (day) within the simulation.
```

Comment:
I'd leave (day) out. The data doesn't need to be based on days (though it currently is). As long as they all agree, the model wouldn't care.